WooCommerce/WooCommerceTests/ViewRelated/Search/Order/OrderSearchUICommandTests.swift

```diff
@@ -0,0 +1,171 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+import Storage
+import Networking
+import protocol WooFoundation.Analytics
+
+final class OrderSearchUICommandTests: XCTestCase {
+    let siteID: Int64 = 12345
+    private var storageManager: MockOrderStatusesStoresManager!
+    private var analyticsProvider: MockAnalyticsProvider!
+    private var analytics: WooAnalytics!
+
+    override func setUpWithError() throws {
+        super.setUp()
+        storageManager = MockOrderStatusesStoresManager()
+        analyticsProvider = MockAnalyticsProvider()
+        analytics = WooAnalytics(analyticsProvider: analyticsProvider)
+    }
+
+    override func tearDownWithError() throws {
+        storageManager = nil
+        analyticsProvider = nil
+        analytics = nil
+        super.tearDown()
+    }
+
+    func testCreateStarterViewControllerReturnsNil() throws {
+        // Given
+        let command = OrderSearchUICommand(siteID: siteID, onSelectSearchResult: { _, _ in }, storageManager: storageManager)
+
+        // When
+        let starterViewController = command.createStarterViewController()
+
+        // Then
+        XCTAssertNil(starterViewController, "Expected createStarterViewController to return nil")
+    }
+
+    func testCreateResultsController() {
```

Comment:
By convention, we tend to summarise the situation/expectation in the test name too:
```suggestion
    func test_createResultsController_returns_results_controller_with_correct_predicate_and_sort_options() {
```

Reply:
Renamed in commit 1b2dc9f to follow convention though it kind of doubles up on the string included in the assert.